lib/content-services/src/lib/agent/services/agent.service.ts

```diff
@@ -16,16 +16,18 @@
  */
 
 import { Injectable } from '@angular/core';
-import { AgentPaging, AgentsApi } from '@alfresco/js-api';
+import { AgentPaging, AgentsApi, AgentWithAvatar } from '@alfresco/js-api';
 import { AlfrescoApiService } from '@alfresco/adf-core';
-import { from, Observable, of } from 'rxjs';
+import { BehaviorSubject, forkJoin, from, Observable, of } from 'rxjs';
+import { switchMap } from 'rxjs/operators';
 
 @Injectable({
     providedIn: 'root'
 })
 export class AgentService {
     private _agentsApi: AgentsApi;
     private _mocked = true;
+    private agentsList = new BehaviorSubject<AgentWithAvatar[]>([]);
```

Comment:
You have renamed here agentsList$ to agents$ https://github.com/Alfresco/alfresco-ng2-components/pull/9956#discussion_r1680504260 but can we rename here as well to just agents?